cli/commands/command.go

```diff
@@ -4,6 +4,7 @@ import (
 	"flag"
 	"github.com/phase2/rig/cli/util"
 	"github.com/urfave/cli"
+	"fmt"
```

Comment:
I thought we were loosely following:

* system packages (alphabetical)
* upstream packages (alphabetical)
* custom packages (alphabetical)

Reply:
oops, will fix.